tests/components/wallbox/test_lock.py

```diff
@@ -47,7 +29,7 @@ async def test_wallbox_lock_class(hass: HomeAssistant) -> None:
 
     with requests_mock.Mocker() as mock_request:
         mock_request.get(
-            "https://api.wall-box.com/auth/token/user",
+            "https://user-api.wall-box.com/users/signin",
```

Comment:
It would be better if the tests mocked the library instead and have the tests in the library test the endpoints.

Ideally Home Assistant doesn't know anything about the endpoints that the library uses

Reply:
For a future PR